test/passing/source.ml.ref

```diff
@@ -2714,6 +2714,7 @@ let f (type a) (x : a t) =
     let (I : a t) = x
 
     (* fail because of toplevel let *)
+
```

Comment:
this sort of change might be a significant step back in cases like this where normal comments are used instead of docstrings

Reply:
I'm not sure I understand. This comments was place right after `x` above in the original source.

Reply:
Oh, my mistake, nevermind then, it gets better in these cases too then.